src/System.IO.Packaging/tests/Tests.cs

```diff
@@ -3489,6 +3489,31 @@ public void T100_String_Create_Read_Star()
             tempGuidName.Delete();
         }
 
+        [Fact]
+        public void SetEmptyPropertyToNull()
+        {
+            using (var ms = new MemoryStream())
+            using (var package = Package.Open(ms, FileMode.Create))
+            {
+                package.PackageProperties.Category = null;
+                package.PackageProperties.ContentStatus = null;
+                package.PackageProperties.ContentType = null;
+                package.PackageProperties.Created = null;
+                package.PackageProperties.Creator = null;
+                package.PackageProperties.Description = null;
+                package.PackageProperties.Identifier = null;
+                package.PackageProperties.Keywords = null;
+                package.PackageProperties.Language = null;
+                package.PackageProperties.LastModifiedBy = null;
+                package.PackageProperties.LastPrinted = null;
+                package.PackageProperties.Modified = null;
+                package.PackageProperties.Revision = null;
+                package.PackageProperties.Subject = null;
+                package.PackageProperties.Title = null;
+                package.PackageProperties.Version = null;
```

Comment:
Anything we can assert here? e.g. that reading back all of these properties reads back null?

Reply:
Added some `Assert.Null` statements for each property